app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,9 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @post = Post.find_by(user_id: params[:user_id])
+  end
```

Comment:
- [ ] <b>REQUIRED:</b> In index of Post controller, we shouldn't be searching post by user_id. besides, that query might return more than one record in case the user has created more than one post and by the looks of the variable you have used, you need one post. and I don't even really get why you need one post on Index(where we need to get the posts that belong to the user). so that's not correct. this is much better.

```ruby
def index
    @user = User.find(params[:id])
    @posts = @user.posts
 end

Reply:
Done! 👍